app/sidekiq/vre/submit1900_job.rb

```diff
@@ -11,10 +13,24 @@ class Submit1900Job
     sidekiq_options retry: RETRY
 
     sidekiq_retries_exhausted do |msg, _ex|
-      Rails.logger.error(
-        "Failed all retries on VRE::Submit1900Job, last error: #{msg['error_message']}"
-      )
-      StatsD.increment("#{STATSD_KEY_PREFIX}.exhausted")
+      monitor = VRE::Monitor.new
+      monitor.track_submission_exhaustion(msg)
+
+      if Flipper.enabled?(:vre_trigger_action_needed_email)
```

Comment:
It would be best to create a method for everything in this ' if Flipper.enabled?' block for readability.